Update syntax-ii.md (#2944)
enkidevs/curriculum: data-analysis/data-tools/data-analysis-using-sql/syntax-ii.md

| 2002 | Harry Potter and the Chamber of Secrets  |
| 2004 | Harry Potter and the Prisoner of Azkaban |

> We can add in multiple `WHERE` clauses using the `AND` keyword. Wrapping "Harry Potter" in `%`, the wildcard character, allows you to pick up all titles that have "Harry Potter" somewhere in the name.

Reviewer: what do you think about this phrasing?

```suggestion
> 💡 We can add in multiple `WHERE` clauses using the `AND` keyword.

We can use the `LIKE` keyword to look for pattern (instead of exact) matches. The filter `WHERE title LIKE "%Harry Potter%"` tells SQL to look for all the movies that have `"Harry Potter"` somewhere in the name.

This pattern matching is achieved with the `%` wildcard character.
```

> > 💡 When using wildcards (`%`, `_`, `[]`, ...) you should substitute the `=` operator with the `LIKE` operator.

Reviewer: we can remove this line if we merge my suggestion

```suggestion
```

> For more information, check out the **Learn More** link to find out how to make `WHERE` less strict by using the keywords `LIKE`, `BETWEEN`, `AND`, `OR` and `IN`, as well as wildcards `%`.

Reviewer: we can trim the phrase a bit

```suggestion
> 💡 Check out the **Learn More** section for more information about the `WHERE` keyword.
```

- WHERE
- `>`
- AND

> - LIKE

Reviewer: we don't need to change this to `LIKE` because the filter is looking at an exact match

```suggestion
- =
```
